src/status_im/chat/styles/message/message.cljs

```diff
@@ -83,9 +83,9 @@
   {:width     36
    :alignSelf :flex-start})
 
-(def photo-view {:borderRadius 12})
+(def photo-view {:borderRadius 18})
 (def photo
-  {:borderRadius 12
+  {:borderRadius 18
```

Comment:
If both are always the same can we have `photo` merge `photo-view` ?

Reply:
done